### ISSUE TITLE
Suppress astropy warnings?

### ISSUE THREAD

**Issue by AtreyeeS** [gammapy/gammapy#3532]
While running the data reduction, I get multiple astropy WARNINGS. This is rather annoying for reducing a large number of observations - the notebooks start to get unreadable. 

Typical warning:
`WARNING: VerifyWarning: Invalid keyword for column 30: Column null option (TNULLn) is invalid for binary table columns of type '1E' (got -1).  The invalid value will be ignored for the purpose of formatting the data in this column. [astropy.io.fits.column]`
or astropy deprecation warnings

I do not want to change the general log level, because I still want to see the Gammapy warnings.
Can we (show an example of how to) suppress these messages?

**Comment by maxnoe** [comment]
To suppress this specific warning, you can do:

```
from astropy.io.fits.verify import VerifyWarning
import warnings

with warnings.catch_warnings():
    warnings.simplefilter('ignore', VerifyWarning)
```

But since it is really wrong in the files, I would not encourage adding this to the documentation. People should complain about it at the source of the fits files.

**Comment by QRemy** [comment]
You can also add the module you want to filter:
```
warnings.filterwarnings(
    action='ignore',
    category=VerifyWarning,
    module='astropy'
)
```

**Comment by maxnoe** [comment]
@QRemy that's good in general yes, but as `VerifyWarning` is a custom warning from the astropy fits code, I would be surprised if anything else is raising this warning.

**Comment by adonath** [comment]
@registerrier I guess an how to entry will do?

**Comment by adonath** [comment]
Fixed by #3583.
